c++/src/Compression.cc

```diff
@@ -232,6 +244,22 @@ namespace orc {
       }
       rawInputBuffer.resize(0);
     }
+  }
+
+  bool CompressionStream::Next(void** data, int* size) {
+    if (rawInputBuffer.size() > compressionBlockSize) {
+      std::stringstream ss;
+      ss << "uncompressed data size " << rawInputBuffer.size() << " is larger than block size "
+         << compressionBlockSize
+         << ". compressionBlockSize should be set equal to multiply of "
```

Comment:
```suggestion
         << ". compressionBlockSize should be set equal to multiple of "
```

Reply:
BTW, we should only put useful error message here. It seems the error happens when the size is greater than the threshold and we should not say anything about the multiple thing. That is something we need to check when creating the compression stream.

Reply:
Make sense, I'll delete those irrelevant comments